### PR TITLE
Download files implementation

### PR DESCRIPTION
**Pull request by alex-deaconu**

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc actually I fixed the issue with the progress bar in the same branch. Thanks.

File: test/e2e/storage/cases/download.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+'use strict';
+var expect = require('rv-common-e2e').expect;
+var HomePage = require('./../../launcher/pages/homepage.js');
+var LoginPage = require('./../../launcher/pages/loginPage.js');
+var CommonHeaderPage = require('rv-common-e2e').commonHeaderPage;
+var helper = require('rv-common-e2e').helper;
+var StorageHomePage = require('./../pages/storageHomePage.js');
+var FilesListPage = require('./../pages/filesListPage.js');
+
+var TrashScenarios = function() {
```
Comment (ezequielc) [inline test/e2e/storage/cases/download.js]: @alex-deaconu Not a big deal, but scenario's name wasn't changed from Trash. Thanks

Comment (alex-deaconu) [inline test/e2e/storage/cases/download.js]: @ezequielc fixed. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu One unit test is failing, so I can't test the staged version. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc sorry about that. Fixed. Also updated folder path as per the widget-test PR. Please review. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu No worries, thanks for fixing. :+1:

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
       "jshint-stylish": "^2.0.1",
       "run-sequence": "^1.1.1",
       "rv-common-e2e": "git://github.com/Rise-Vision/rv-common-e2e.git#0.0.4",
-      "widget-tester": "git://github.com/Rise-Vision/widget-tester.git#1.6.3"
+      "widget-tester": "git://github.com/Rise-Vision/widget-tester.git#feature/download-folder"
```
Comment (ezequielc) [inline package.json]: @alex-deaconu Just a reminder to change here after merging widget-tester. Cheers

Comment (alex-deaconu) [inline package.json]: @ezequielc you're right, thanks for the reminder ;)

**Pull request merged**